packages/common/test/libs.test.ts

```diff
@@ -217,11 +217,11 @@ describe("Module `arcgis-html-sanitizer`: ", () => {
     });
 
     it("tests XSS cases", () => {
-      console.log(
-        "Running " +
-          xssFilterEvasionTestCases.testCases.length +
-          " XSS test cases"
-      );
+      // console.log(
```

Comment:
Just to know, this is here because I swap out the basic tests with a more complete set of tests, and this lets me verify that I'm testing with 143 cases instead of just the basic 6. But it does insert an untidiness in the output, so I'll remove it.